Examples/ObjectiveC/RainFallExample.m

```diff
@@ -0,0 +1,140 @@
+#import "RainFallExample.h"
```

Comment:
Let's rename this to something like `AnimatedFillLayer`. We want example names to be a little more generic. This helps users find the techniques that work for their use case.